hooks/pre-commit

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+#
+# Recommended pre-commit git hook for activity-stream github repo
+#
+# Based on the example script in the prettier docs at
+# https://prettier.io/docs/en/precommit.html
+#
+FILES=$(git diff --cached --name-only --diff-filter=ACM "*.js" "*.jsx" | sed 's| |\\ |g')
```

Comment:
We'll want to include .jsm and not sure if we really need to fix up files with spaces as we don't have any

Reply:
Yep, will fix the JSM thing. 

I'm inclined to leave the space fixer since it makes the script more robust in case someone adds a file with space.